src/Caching/Memory/src/Microsoft.Extensions.Caching.Memory.csproj

```diff
@@ -8,6 +8,10 @@
     <PackageTags>cache;memorycache</PackageTags>
   </PropertyGroup>
 
+  <ItemGroup>
+    <InternalsVisibleTo Include="Microsoft.Extensions.Caching.Memory.Tests" />
+  </ItemGroup>
```

Comment:
This is nice, we should start using this in Razour.

Reply:
Yeah, I greatly prefer this to adding AssemblyInfo files which have to manually copy the key.